css/styles.css

```diff
@@ -444,15 +444,15 @@ tr:nth-child(odd) {
 }
 .logo-link {
     display: inline-block;
-    width: 220px;
-    height: 64.77px;
-    background: url("/images/logo.png");
-    background-size: 217px auto;
+    width: 117px;
+    height: 65px;
+    background: url("/logos/kafka_logo--simple.png");
+    background-size: auto 65px;
     background-repeat: no-repeat;
     /* small: 168 x 50 */
 }
 .page-dark .logo-link {
-    background: url('/logos/kafka-white.svg');
```

Comment:
Should we remove the old logo image file?

Reply:
I've cleaned up all the "dark theme" stuff, which included any of the white logos that were in use.